facades/PC/README.md

```diff
@@ -0,0 +1,99 @@
+# PC Facade
+
+The _PC Facade_ is the front-facing layer for running _Terasology_ on the PC.
```

Comment:
```suggestion
The _PC Facade_ is the front-facing layer for running _Terasology_ on a PC.
```
(Although this change might be a personal preference.)
It might also be worth mentioning that "PC" (Personal Computer?) refers to the Windows, macOS and Linux platforms specifically.

Reply:
@BenjaminAmos How about the following phrasing?
```suggestion
The _PC Facade_ is the front-facing layer for running _Terasology_ on a personal computer (PC) with a Windows, macOS, or Linux operating system.
```

Reply:
I've changed my mind about expanding the personal computer abbreviation. I don't think any changes are necessary here anymore. New OSes might come along and we're not doing too much platform-specific stuff ourselves (I think). Terasology's supported platforms are well documented elsewhere.

```suggestion
The _PC Facade_ is the front-facing layer for running _Terasology_ on computers running Windows, macOS, or Linux.
```